index.js

```diff
@@ -25,65 +26,74 @@ function getRandomNumber(min, max) {
 function checkGuess() {
   // Get value from guess input element
   const guess = parseInt(guessInput.value, 10);
-  attempts = attempts + 1;
+  attempts += 1;
 
   hideAllMessages();
 
   if (guess === targetNumber) {
     numberOfGuessesMessage.style.display = '';
     numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
-
     correctMessage.style.display = '';
-
     submitButton.disabled = true;
     guessInput.disabled = true;
+    resetButton.style.display = '';
+
   }
 
   if (guess !== targetNumber) {
     if (guess < targetNumber) {
       tooLowMessage.style.display = '';
     } else {
-      tooLowMessage.style.display = '';
+      tooHighMessage.style.display = '';
     }
 
     const remainingAttempts = maxNumberOfAttempts - attempts;
 
     numberOfGuessesMessage.style.display = '';
     numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
+    resetButton.style.display = '';
   }
 
-  if (attempts ==== maxNumberOfAttempts) {
+  if(guess > maxNumberOfAttempts){
+    alert `Remember: is 1 to 99`;
+    resetButton.style.display = '';
+  }
```

Comment:
It's great that you tried to show a warning when the number is out of bounds, Marilyn! But you're comparing it wrong. You don't have to test that the guess is greater than the max number of attempts, you have to test that it is not less than 1 and not greater than 99.